dir.props

```diff
@@ -125,7 +125,9 @@
 
   <!-- Create a collection of all project.json files for dependency updates. -->
   <ItemGroup>
-    <ProjectJsonFiles Include="$(SourceDir)**/project.json" />
+    <!-- Exclude PackageFiles tool-runtime and test-runtime project.json files. -->
+    <ProjectJsonFiles Include="$(SourceDir)**/project.json"
+                      Exclude="$(SourceDir)Microsoft.DotNet.Build.Tasks\PackageFiles\**\project.json" />
```

Comment:
@dagood I can definitely see msbuild being picky about but what I don't understand is that I wouldn't expect Include and Exclude to be different. We are doing wildcard in both places and so if it is broken I would expect it to be broken by one or the other here. At any rate just be sure to test this and I'm OK with it. I wouldn't be surprised if this is different on non-windows as well, but for BuildTools in particular we don't build on anything except windows right now.